sentinelhub/api/batch_statistical.py

```diff
@@ -0,0 +1,148 @@
+"""
+Module implementing an interface with
+`Sentinel Hub Batch Processing API <https://docs.sentinel-hub.com/api/latest/api/batch-statistical/>`__.
+"""
+import datetime as dt
+import logging
+from dataclasses import dataclass, field
+from typing import Any, Optional, Union
+
+from dataclasses_json import CatchAll, LetterCase, Undefined
+from dataclasses_json import config as dataclass_config
+from dataclasses_json import dataclass_json
+
+from ..type_utils import Json, JsonDict
+from .batch_base import BaseBatchClient, BaseBatchRequest, BatchRequestStatus, BatchUserAction
+from .utils import datetime_config, enum_config, remove_undefined
+
+LOGGER = logging.getLogger(__name__)
+
+BatchStatisticalRequestType = Union[str, dict, "BatchStatisticalRequest"]
+
+
+class SentinelHubBatchStatistical(BaseBatchClient["BatchStatisticalRequest"]):
+    """An interface class for Sentinel Hub Batch Statistical API
+
+    Check `Batch Statistical API <https://docs.sentinel-hub.com/api/latest/reference/#tag/batch_statistical>`__ for more
+    information.
+    """
+
+    # pylint: disable=too-many-public-methods
+    @staticmethod
+    def _get_service_url(base_url: str) -> str:
+        """Provides URL to Batch Statistical API"""
+        return f"{base_url}/api/v1/statistics/batch"
+
+    def create(
+        self,
+        input_data: JsonDict,
```

Comment:
`input_data` should actually have a type `List[JsonDict]`. In the process and statistical API interfaces we even use type `List[Union[JsonDict, InputDataDict]]` for this so maybe this an even better type.

Reply:
done